acceptance-test/user-pathways/upload-file/upload-file.test.js

```diff
@@ -19,7 +19,7 @@ const {
 describe('Upload File(s)', () => {
 
     it('upload 1 document', async() => {
-        const { browser, page, hostIP: APP_CONTAINER_HOST } = await bootstrap.buildBrowser();
+        const { page, hostIP: APP_CONTAINER_HOST } = await bootstrap.buildBrowser();
 
         const { env: { PORT } } = process;
```

Comment:
O I missed this from before, what's going on here? I know it's related to your other PR

Reply:
Do you mean the destructuring of the returned object?
This is another way of writing:
const myBrowser = await bootstrap.buildBrowser();
const browser = myBrowser.browser;
const page = myBrowser. page;
const APP_CONTAINER_HOST = myBrowser. hostIP;

or

What the line achieves?
Uses puppeteer to create a connection with the container in which the browser lies and returns the resulting browser object (The browser which puppeteer controls), the page object (the page which is now open as a result of the browser opening) and the hostIP for the browser being run in Chrome Dev tools for use within the URL constructed later on in the script.

Let me know if that doesn't answer the question you had.

Reply:
it was more this one
`const { env: { PORT } } = process;` and what it achieved.  Is it just adding `Process.env.PORT` to `process`?